detect_secrets/core/baseline.py

```diff
@@ -34,13 +34,19 @@ def initialize(
         exclude_lines=exclude_lines_regex,
     )
 
-    if os.path.isfile(path):
```

Comment:
Another reason for the original logic was that it allowed us to do:

```
detect-secrets scan test_data/files/file_with_secrets.py
```

as well as

```
detect-secrets scan test_data/
```

using `_get_git_tracked_files` to get files, if it is a directory.

How does this change handle that?

Reply:
Should be working now :)